orca-core/src/main/groovy/com/netflix/spinnaker/orca/ExecutionStatus.groovy

```diff
@@ -95,6 +95,7 @@ enum ExecutionStatus {
   final ExitStatus exitStatus
 
   private static final Collection<ExecutionStatus> SUCCESSFUL = [SUCCEEDED, STOPPED, SKIPPED]
+  private static final Collection<ExecutionStatus> FAILURE = [TERMINAL, STOPPED, FAILED_CONTINUE]
```

Comment:
Thoughts on if `STOPPED` should result in the auto cancel behavior?

I was inclined to say yes (_because that's how the pipeline in question that exposed the issue is setup_) ... also STOPPED is an indication of a failure that was configured to not halt the overall pipeline.